tap_azure_git/__init__.py

```diff
@@ -1150,7 +1150,8 @@ def get_build_timeline(schema, org, repo_path, build, state, mdata, start_date):
         build_timeline = {
             **raw_build_timeline,
             '_sdc_repository': sdc_repository,
-            '_sdc_id': '{}/build/{}/timeline/{}'.format(sdc_repository, build['id'], raw_build_timeline['id'])
+            '_sdc_id': '{}/build/{}/timeline/{}'.format(sdc_repository, build['id'], raw_build_timeline['id']),
+            'build_id': build['id']
```

Comment:
@myleshenderson also thinking of adding `_sdc_build_id` which would be `'{}/build/{}'.format(sdc_repository, build['id'])`.  

Azure dev ops is kind of weird because `id` is not globally unique.  You could join on `_sdc_build_id` or `_sdc_repository` and `build_id`, not sure which is preferrable.